### ISSUE TITLE
Add aria-haspopup and more descriptive aria-label to hotgraphic pin

### ISSUE THREAD

**Issue by amygroshek** [adaptlearning/adapt_framework#2051]
### Add aria-haspopup and more descriptive aria-label to hotgraphic pin
Improves accessibility by adding screenreader popup notification and aria-label specific to the hotgraphic pin functionality

### Your environment
* 2.0.6
* All/markup
* Macbook pro 2012, MacOS High Sierra

### Steps to reproduce
Add a hotgraphic to content and `grunt build`. Access the component with a screenreader. 

### Expected behaviour
When the screenreader focus moves to the pin, the user should be notified that there is a selectable item that launches a dialog popup.

### Actual behaviour
At present the screenreader reads something like "Item #: Item title". There is no popup notification.

See https://github.com/adaptlearning/adapt-contrib-hotgraphic/pull/150
